app/helpers/textile_helper.rb

```diff
@@ -1,10 +1,10 @@
 module TextileHelper
```

Comment:
Is this the right file and module name?

Reply:
I think the urls it generates are a good sense of direction for naming: `markup_*`